manifests/init.pp

```diff
@@ -8,6 +8,10 @@
   Float   $cvmfs_quota_ratio                                     = 0.85,
   Stdlib::Absolutepath $cvmfs_cache_base                         = '/var/lib/cvmfs',
   Optional[Enum['yes','no']] $cvmfs_claim_ownership              = undef,
+  Optional[Hash[Variant[Integer,String], Integer, 1]]
```

Comment:
Just write on one line and push all the `=` right.

Reply:
Done, and also fixed a typo in the new documentation :smile: .